src/test/java/org/folio/marccat/resources/AuthorityRecordTest.java

```diff
@@ -42,13 +48,26 @@ public void getDocumentCountById() throws IOException {
     String templateJson = IOUtils.toString(this.getClass().getResourceAsStream("/authority/name.json"),
         String.valueOf(StandardCharsets.UTF_8));
 
-    given().headers("Content-Type", "application/json").headers(headers).queryParam("view", "-1")
-        .queryParam("lang", "eng").body(templateJson).when().post(url).then().statusCode(201);
+    given().headers(CONTENT_TYPE, FILE_TYPE).headers(headers).queryParam("view", "-1").queryParam("lang", "eng")
+        .body(templateJson).when().post(url).then().statusCode(201);
 
     url = getURI("/marccat/document-count-by-id");
     headers = addDefaultHeaders(url, StorageTestSuite.TENANT_ID);
 
     given().param("id", "1").param("view", "-1").headers(headers).when().get(url).then().statusCode(200);
   }
 
+  @Test
+  public void getAuthorityFixedFieldDisplayValue() throws IOException {
+
+    String url = getURI("/marccat/authority-record/fixed-field-display-value");
+    Map<String, String> headers = addDefaultHeaders(url, StorageTestSuite.TENANT_ID);
+    String templateJson = IOUtils.toString(this.getClass().getResourceAsStream("/authority/fixedField.json"),
+        String.valueOf(StandardCharsets.UTF_8));
+    ObjectMapper objectMapper = new ObjectMapper();
```

Comment:
Just curious why you Unmarshall the json here but not in the other methods...

Reply:
Because in this case, I have to send the diferent values to de UI